docs/dogfood/index.md

```diff
@@ -0,0 +1,143 @@
+3. Run `make bin`
+    
+    <aside>
+    💡 If you run into the following error:
+    
+    ```js
+    pg_dump: server version: 13.7 (Debian 13.7-1.pgdg110+1); pg_dump version: 11.16 (Ubuntu 11.16-1.pgdg20.04+1)
+    pg_dump: aborting because of server version mismatch
+    ```
+    
+    Don’t fret! This is a known issue. To get around it:
+    
+    1. Add `export DB_FROM=coderdb` to your `.bashrc` (make sure you `source ~/.bashrc`)
+    2. Run `sudo service postgresql start`
+    3. Run `sudo -u postgres psql` (this will open the PostgreSQL CLI)
+    4. Run `postgres-# alter user postgres password 'postgres';`
+    5. Run `postgres-# CREATE DATABASE coderdb;`
+    6. Run `postgres-# grant all privileges on database coderdb to postgres;`
+    7. Run `exit` to exit the PostgreSQL terminal
+    8. Try `make bin` again.
+    </aside>
+    
+4. Run `./scripts/develop.sh` which will start *two* separate processes:
+    1. `[http://localhost:3000](http://localhost:3000)` — backend API server 👈 Backend devs will want to talk to this
+    2. `[http://localhost:8080](http://localhost:8080)` — Node.js dev server 👈 Frontend devs will want to talk to this
+    
+    <aside>
+    💡 If you’re dogfooding on v1, add a DevURL to port 8080, HTTP
+    If you’re dogfooding on v2, either:
+    - Use `coder tunnel 8080 8080` 
+    - Use VSCode’s built-in port forwarding. See: [https://code.visualstudio.com/docs/remote/ssh](https://code.visualstudio.com/docs/remote/ssh)
+    
+    </aside>
+    
+5. Ensure that you’re logged in: `./scripts/coder-dev.sh list` — should return no workspace. If this returns an error, double-check the output of running `scripts/develop.sh`.
+6. A template named `docker-amd64` (or `docker-arm64` if you’re on ARM) will have automatically been created for you. If you just want to create a workspace quickly, you can run `./scripts/coder-dev.sh create myworkspace -t docker-amd64` and this will get you going quickly!
+7. To create your own template, you can do: `./scripts/coder-dev.sh templates init` and choose your preferred option. 
+For example, choosing “Develop in Docker” will create a new folder `docker` that contains the bare bones for starting a Docker workspace template.
+Then, enter the folder that was just created and customize as you wish.
+    
+    <aside>
+    💡 **For all Docker templates:**
+    This step depends on whether you are developing on a Coder v1 workspace, versus a Coder v2 workspace, versus a VM, versus locally. In any case, check the output of the command `docker context ls` to determine where your Docker daemon is listening. Then open `./docker/main.tf` and check inside the block  `provider "docker"` that the variable `"host"` is set correctly.
+    
+    </aside>
+
+## Troubleshooting 
+
+### What if I'm having trouble because I have the old coder CLI installed?
+
+Move `coder` to `coderv2`:
+```shell
+sudo mv coder /usr/local/bin/coderv2
+coderv2 config-ssh
+```
```

Comment:
Ideally we wouldn't have both side-by-side but I do this locally for my own convenience.